index.js

```diff
@@ -54,10 +53,8 @@ module.exports = function(source, map, meta) {
 
       replaceSource.replace(
         match.index,
-        pattern.lastIndex,
-        forPath
-          ? JSON.stringify(addonRequest)
-          : `require(${JSON.stringify(addonRequest)})`,
+        pattern.lastIndex - 1,
```

Comment:
This is a bit interesting: `lastIndex` is the correct *exclusive* index to end the replacement at (the end of the last match), it's that for some bizarre reason `ReplaceSource` decided that their end parameter would be *inclusive* (probably to do with location tracking being more comprehensible).
I'll fix this with a comment even if this doesn't go in.